web-extensions/chrome/engine/datamodel.js

```diff
@@ -597,6 +597,76 @@ BmcDataAPI.prototype.unregisterComic = function(comicId, cb) {
     });
 };
 
+
+/**
+ * @callback BmcDataAPI~unaliasCb
```

Comment:
What is this supposed to document exactly? :-/

Reply:
The exact prototype of the expected callback , defined as the type "BmcDataApi~unaliasCb". ("`" is an equivalent of scope operator in this syntax).